src/main/java/ledger/user_interface/ui_controllers/component/tablecolumn/AccountColumn.java

```diff
@@ -24,11 +32,51 @@ public AccountColumn() {
         this.initController(pageLoc, this, "Unable to load AccountColumn");
     }
 
+    ObservableList<Account> observableAllAccounts;
+
     @Override
     public void initialize(URL location, ResourceBundle resources) {
+        TaskWithReturn<List<Account>> getAllAccountTask = DbController.INSTANCE.getAllAccounts();
+        getAllAccountTask.startTask();
+        List<Account> allAccounts = getAllAccountTask.waitForResult();
+        this.observableAllAccounts = FXCollections.observableList(allAccounts);
+
+
         this.setCellValueFactory(new PropertyValueFactory<Transaction, Account>("account"));
-        this.setCellFactory(TextFieldTableCell.forTableColumn(new AccountStringConverter()));
+        this.setCellFactory(ComboBoxTableCell.forTableColumn(new AccountStringConverter(), observableAllAccounts));
+        this.setOnEditCommit(this.accountEditHandler);
         this.setComparator(new AccountComparator());
-        this.setEditable(false);
+
+        DbController.INSTANCE.registerPayyeeSuccessEvent(this::updateAccountList);
```

Comment:
I believe this should be registerAccountSuccessEvent(). This feature also doesn't work. After adding a new account, it is not reflected int he Account column dropdown options.